<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>771</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>设置</string>
    </property>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>十六进制发送</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>十六进制接收</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;端口&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PortlineEdit">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>60000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ListenpushButton">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>30</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Listen</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>40</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;本机IP&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LocalIPlineEdit">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>30</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>工作界面</string>
    </property>
    <widget class="QLabel" name="ConnectedHostlabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;已连接主机&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="SendlineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>329</y>
       <width>631</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="SendpushButton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>330</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>SEND</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ConnectedHostcomboBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>290</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="TexttextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>741</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ClearpushButton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>282</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>CLEAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>564</x>
       <y>282</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Cutoff Socket</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>209</y>
       <width>741</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(85, 0, 0);</string>
     </property>
     <property name="title">
      <string>待发送主机</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;待发送主机&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="subConnectingComboBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>371</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sendToConnectingBtn">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;宋体&quot;;</string>
      </property>
      <property name="text">
       <string>Link</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>741</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>待接收主机</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;待接收主机&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="recConnectingComboBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="recConnectingBtn">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;宋体&quot;;</string>
      </property>
      <property name="text">
       <string>Link</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
